.travis.yml

```diff
@@ -46,9 +52,12 @@ before_install:
   - export MVN_HOME=$HOME/apache-maven-3.6.1
   - if [ ! -d $MVN_HOME/bin ]; then wget https://archive.apache.org/dist/maven/maven-3/3.6.1/binaries/apache-maven-3.6.1-bin.tar.gz -P $HOME; tar xzvf $HOME/apache-maven-3.6.1-bin.tar.gz -C $HOME; fi
   - export PATH=$MVN_HOME/bin:$PATH
+
 install: /bin/bash ./dev-tools/travis/travis-install.sh `pwd`
 script:
-  - /bin/bash ./dev-tools/travis/travis-script.sh `pwd` $MODULES
+  - if [[ $(uname -m) != 's390x' ]]; then 
```

Comment:
Why not build all modules on s390x as well?

Reply:
Hi @kishorvpatil , 
"travis-script.sh" contains tests for each of the MODULES and for some of the tests rocksdb is required. Currently there is no support for rocksdb for s390x. Found a PR raised to add s390x support to rocksdb ([link](https://github.com/facebook/rocksdb/pull/6168)) , waiting for it to be merged.

Reply:
Travis build has been failing. Might be related to this

Reply:
@Ethanlm all builds have passed now.